app/views/admin/edition_legacy_associations/edit.html.erb

```diff
@@ -1,28 +1,20 @@
 <% page_title "Edit legacy associations: " + @edition.title %>
 <div class="row">
   <h1><%= @edition.title %></h1>
+  <p>
+    Policies, policy areas and specialist sectors will continue to be supported until the topic taxonomy replaces them.
```

Comment:
Perhaps capitalise "Topic Taxonomy" here, as I see it as a proper noun. Same with "Policy Areas" and "Specialist Sectors" actually.

Reply:
👍